study-builder/studies/angio/study.conf

```diff
@@ -9,7 +9,7 @@
         "guid": "cmi"
     },
     "study": {
-        "name": "Angiosarcoma Project",
+        "name": "ascproject",
```

Comment:
Just restoring the names in the config files. Note that we DID NOT push this `name` change to production when the file was changed, so we're just revert the change here.